README.md

```diff
@@ -24,6 +24,69 @@ poetry add git+https://github.com/aai-institute/nnbench.git
 
 A PyPI package release is planned for the future.
 
+## Usage Example
+
+An example repository to benchmark a ML model contains three files. The `training.py` contains the code to train the model, the `benchmark.py` contains the benchmark functions, and the `run_benchmark.py` the benchmark execution logic. 
```

Comment:
For this example it's fine, but I'm not sure if we want to advertise that file structure regardless.

It makes sense if we think about benchmarks in a MLOps pipeline way. Question is, do we still include it in the introductory example to only showcase the basic application?

I think this example should serve as the Quickstart document, and be a single-file monolith to convey the simplicity of the package.

Then, if the user is interested, we link to user guides (to be written) on file structure/organization, tagging, subclassing runners,reporters,etcpp. What do you think?